app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,19 @@
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <h2>Resend confirmation instructions</h2>
+
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Resend confirmation instructions", class: "btn btn-primary" %>
+  </div>
+<% end %>
```

Comment:
- [ ] Great job working on the category so far but I recommend to change the color of the buttons as well here as mentioed in the requirements so I highly recommend to change it 💪🏻🔥


## the nav is blue and the button bg is green , here's the main colors of the project

![image](https://github.com/Timbar09/Budget-App/assets/58553711/ac1d1402-566b-4b04-a25e-e0cdcb24640a)


- Your
![image](https://github.com/Timbar09/Budget-App/assets/58553711/c9fd123d-2050-46dd-bf84-c1998e97c793)


- Given
![image](https://github.com/Timbar09/Budget-App/assets/58553711/d41d4890-390c-417a-95a9-a5f8cec946da)